ref-exchange/build_docker.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Exit script as soon as a command fails.
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+docker run \
+     --mount type=bind,source=$DIR/..,target=/host \
+     --cap-add=SYS_PTRACE --security-opt seccomp=unconfined \
+     -w /host/ref-exchange \
+     -e RUSTFLAGS='-C link-arg=-s' \
+     nearprotocol/contract-builder \
+     /bin/bash -c "rustup target add wasm32-unknown-unknown; cargo build --target wasm32-unknown-unknown --release"
```

Comment:
Every subsequent run will create a new container, without deleting the previous one. 
Ideally we would like to create a container once and then reuse it. Could you update this please?